kong/tools/utils.lua

```diff
@@ -783,6 +783,29 @@ _M.format_host = function(p1, p2)
   end
 end
 
+local CONTROLS = [[\x00-\x1F\x7F]]
+local HIGHBIT = [[\x80-\xFF]]
+local SEPARATORS = [==[ \t()<>@,;:\\\"\/?={}\[\]]==]
+local HTTP_TOKEN_FROBID_PATTERN = "[".. CONTROLS .. HIGHBIT .. SEPARATORS .. "]"
```

Comment:
```suggestion
local HTTP_TOKEN_FORBID_PATTERN = "[".. CONTROLS .. HIGHBIT .. SEPARATORS .. "]"
```
Also references need to be fixed.